c++/src/kj/compat/brotli-test.c++

```diff
@@ -122,13 +123,17 @@ public:
   }
 
   Promise<void> write(const void* buffer, size_t size) override {
+    writes.add(size);
     bytes.addAll(arrayPtr(reinterpret_cast<const byte*>(buffer), size));
     return kj::READY_NOW;
   }
   Promise<void> write(ArrayPtr<const ArrayPtr<const byte>> pieces) override {
+    size_t total = 0;
     for (auto& piece: pieces) {
+      total += piece.size();
```

Comment:
Doesn't summing up the writes here prevent us from verifying the correct behavior here? We try to see that if we compress several pieces we only flush once, but we add up all the individual writes here. Replacing the changes in this function with just `writes.add(size);` in this line should work. Same applies for gzip-test

Reply:
This ends up not even being used in the modified test, given that BrotliAsyncOutputStream/GzipAsyncOutputStream only ever actually call the `write(buffer, size)` variant of the function. Strictly speaking I could omit this change entirely but included it for completeness, treating it as a single write rather than multiple writes.